h/templates/home.html.jinja2

```diff
@@ -0,0 +1,305 @@
+{% set rss_feed_url = base_url + "feed/" %}
+
+{% extends "h:templates/layouts/base.html.jinja2" %}
+
+{% block page_title %}Hypothesis | The Internet, peer reviewed.{% endblock %}
+
+{% block meta %}
+  {{ super() }}
+  <link rel='canonical' href='{{ base_url }}'>
+  <link href="{{ rss_feed_url }}" rel="alternate" type="application/rss+xml">
+{% endblock %}
+
+{% block styles %}
+  {% assets "front_page_css" %}
+      <link rel="stylesheet" href="{{ ASSET_URL }}">
+  {% endassets %}
+{% endblock %}
+
+{% block content %}
+  <header class="banner hiring">
+    <div class="container">
+      <span>
+        We're hiring
+        <a href="{{ base_url }}jobs/#developer">developers</a>.
```

Comment:
Is there a good reason to have all these links as `{{ base_url }}...`? Can't they just be relative links?